tests/phpunit/integration/Core/Conversion_Tracking/Conversion_TrackingTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Conversion_TrackingTest
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Tests\Core\Conversion_Tracking;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Conversion_Tracking\Conversion_Events_Provider;
+use Google\Site_Kit\Core\Conversion_Tracking\Conversion_Tracking;
+use Google\Site_Kit\Tests\Core\Conversion_Tracking\Conversion_Event_Providers\FakeConversionEventProvider;
+use Google\Site_Kit\Tests\Core\Conversion_Tracking\Conversion_Event_Providers\FakeConversionEventProvider_Active;
+use Google\Site_Kit\Tests\TestCase;
+
+class Conversion_TrackingTest extends TestCase {
+
+	/**
+	 * Conversion_Tracking instance.
+	 *
+	 * @var Conversion_Tracking
+	 */
+	private $conversion_tracking;
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->conversion_tracking = new Conversion_Tracking( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
+
+		$this->conversion_tracking::$conversion_event_providers = array(
```

Comment:
I know that technically it is possible to access a static property using an instance of the class, but this is not how static properties are supposed to work. Let use the class name:
```suggestion
		Conversion_Tracking::$conversion_event_providers = array(
```